Tasks/ServiceFabricDeploy/utilities.ps1

```diff
@@ -185,12 +185,12 @@ function Add-Certificate
 
         if ($ConnectedServiceEndpoint.Auth.Parameters.CertificatePassword)
```

Comment:
If certificatePassword is not present, should we throw now? Else Import can have improper certPassword.